src/commonMain/kotlin/exchange.dydx.abacus/state/app/helper/TriggerOrderToastGenerator.kt

```diff
@@ -119,13 +119,13 @@ class TriggerOrderToastGenerator(
         }
         val takeProfitOrders = triggerOrders?.filter { order ->
             selectedSubaccountPosition?.side?.current?.let { currentSide ->
-                (order.type == OrderType.takeProfitMarket || order.type == OrderType.takeProfitLimit) &&
+                (order.type == OrderType.takeProfitMarket || order.type == OrderType.takeProfitLimit) && order.side != null &&
                     order.side.isOppositeOf(currentSide)
             } ?: false
         }
         val stopLossOrders = triggerOrders?.filter { order ->
             selectedSubaccountPosition?.side?.current?.let { currentSide ->
-                (order.type == OrderType.stopMarket || order.type == OrderType.stopLimit) &&
+                (order.type == OrderType.stopMarket || order.type == OrderType.stopLimit) && order.side != null &&
```

Comment:
could we simplify this to `order.side?.isOppositeOf(currentSide)`?

Reply:
um kinda! can change to `order.side?.isOppositeOf(currentSide) == true`